package.json

```
@@ -1,16 +1,14 @@
 {
   "name": "@helix-design-system/helix-react",
-  "version": "1.0.0-alpha.4",
+  "version": "1.0.0-alpha.5",
```

You can revert this change.  The version will get bumped when we publish the next npm version.

---

Cool. Some time when your free id like to learn more about how the release part happens.
```suggestion
  "version": "1.0.0-alpha.4",
```